web-client/src/components/DashboardView/DashboardView.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import InputtingForm from "./InputtingForm";
+import "./DashboardViewStyle.css";
+
+class DashboardView extends Component {
+
+  render() {
+    return (
+      <div>
+        <iframe
+          className="result-box"
+          title="Result box"
+          srcDoc="<i>Result...</i>"
+          name="iframe-result"
+        />
+
+        <div style={{ padding: 16 }}>
+          <InputtingForm
+            formMethod="POST"
+            apiUrl="http://localhost:3000/api/files/path?_method=PUT"
```

Comment:
It will not work if accessing from outside of the server.

Reply:
I create issue #52  to resolve this one

Reply:
And issue #51 for writing unit tests